syncd/FlexCounter.cpp

```diff
@@ -31,6 +31,13 @@ static const std::string ATTR_TYPE_PG = "Priority Group Attribute";
 static const std::string ATTR_TYPE_MACSEC_SA = "MACSEC SA Attribute";
 static const std::string ATTR_TYPE_ACL_COUNTER = "ACL Counter Attribute";
 
+// --> Needed for agg voq stats
+static sai_queue_api_t *sai_queue_api = NULL;
```

Comment:
no api should be uses, VendorCalss shoud be used

Reply:
Sorry, I do not understand. I need a handle `sai_queue_api` so that I can query queue stats like this ` status = sai_queue_api->get_queue_attribute(rid, 1, &attr);`. I am not aware of an alternative to this. What is VendorClass? How to use it? Can you point me to an example?

Reply:
use VendorSai class

Reply:
Done.